frontend/server/src/test/java/com/amazonaws/ml/mms/ModelServerTest.java

```diff
@@ -467,31 +468,35 @@ private void testPredictionsValidRequestSize(Channel channel) throws Interrupted
         Assert.assertEquals(httpStatus, HttpResponseStatus.OK);
     }
 
-    private void loadNoopConfigTests(Channel channel) throws InterruptedException {
+    private void loadTests(Channel channel, String url, String modelName)
+            throws InterruptedException {
         result = null;
         latch = new CountDownLatch(1);
-        HttpRequest req =
-                new DefaultFullHttpRequest(
-                        HttpVersion.HTTP_1_1,
-                        HttpMethod.POST,
-                        "/models?url=noop-v1.0-config-tests&model_name=noop-config&initial_workers=1&synchronous=true");
+        String uri =
+                "/models?url="
```

Comment:
Nitpick, why not StringBuilder?

Reply:
I didn't choose StringBuilder because this is a finite set of strings that need to be concatenated . I think internally the compiler would convert this to a StringBuilder.